pipeline/deploy.sh

```diff
@@ -29,23 +29,26 @@ then
 fi
 
 # Borrow a testbed from CSI Testbed Pool Svc.
-if ! testbed=$(curl -X 'PUT' "${CNS_TESTBEDPOOL_SVC_URL}" -H 'accept: application/json' -u "${CNS_MANAGER_USERNAME}:${CNS_MANAGER_PASSWORD}");
-then
-	echo "Unable to borrow a testbed."
+if  ! testbed=$(curl -X 'PUT'  "$TESTBED_POOL_SERVICE_ENDPOINT/v1/pool/$TESTBED_POOL_ID/borrowTestbed"  -H 'accept: application/json' -u "$CNS_MANAGER_USERNAME:$CNS_MANAGER_PASSWORD"); then
+    echo "Unable to borrow a testbed"
 	exit 1
+else
+    # borrowTestbed API succeed
+    echo "Got a testbed from testbed pool service"
 fi
 
 echo "TestbedInfo: $testbed ."
```

Comment:
Would this log sensitive information from the testbed like passwords?

Reply:
This was logging the testbed details for the users to debug in case of failures. 
Removed logging the testbed details, and we may now use the kubeconfig content from artifacts.